widget/view.controller.js

```diff
@@ -60,7 +60,7 @@
             })
             for (const key in $scope.entity.fields) {
                 if (!selectedFields.includes(key) && !excludeArray.includes(key) && $scope.entity.fields[key].type !== 'manyToMany'
-                   && $scope.entity.fields[key].type !== 'oneToMany') {
+                   && $scope.entity.fields[key].type !== 'oneToMany' && $scope.entity.fields[key].type !== null) {
```

Comment:
jus add a check in the beginning, "$scope.entity.fields[key].type" this will validate if the type is not null or not undefined, later you can explicitly check for null,
(!selectedFields.includes(key) && !excludeArray.includes(key) && $scope.entity.fields[key].type !== 'manyToMany'
                   && $scope.entity.fields[key].type !== 'oneToMany' && $scope.entity.fields[key].type && $scope.entity.fields[key].type !== null)
Like this

Reply:
Done